src/main/java/io/jenkins/plugins/gitlabbranchsource/Cause/GitLabMergeRequestNoteData.java

```diff
@@ -0,0 +1,55 @@
+package io.jenkins.plugins.gitlabbranchsource.Cause;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.gitlab4j.api.webhook.NoteEvent;
+import org.kohsuke.stapler.export.Exported;
+import org.kohsuke.stapler.export.ExportedBean;
+
+import static io.jenkins.plugins.gitlabbranchsource.Cause.GitLabCauseUtils.defaultIntString;
+import static io.jenkins.plugins.gitlabbranchsource.Cause.GitLabCauseUtils.defaultVisibilityString;
+import static org.apache.commons.lang.StringUtils.defaultString;
+
+@ExportedBean
+public class GitLabMergeRequestNoteData {
+
+    private final Map<String, String> variables = new HashMap<>();
+
+    public GitLabMergeRequestNoteData(NoteEvent noteEvent) {
```

Comment:
It's great to have some useful environment variables. But I have two questions about it.

* How users can know the variable name and its meaning
* Do we need so many variables? Or they're all?

Reply:
Answering all comments above here.  Most of variables were copied from another classes from cause package. M bad, meh.
For sure, GITLAB_COMMENT_TRIGGER is enough..